src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1319,10 +1335,13 @@ class ApplicationManagerImpl
    * @brief Starts specified navi service for application
    * @param app_id Application to proceed
    * @param service_type Type of service to start
-   * @return True on success, false on fail
+   * @param params configuration parameters specified by mobile
+   * @return True if service is immediately started or configuration
+   * parameters are sent to HMI, false on other cases
    */
   bool StartNaviService(uint32_t app_id,
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/2af4e80da9542bdd1a149ffeee0b7844a7216453.